src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,46 @@
+package lotto.controller;
+
+import lotto.constant.LottoPrize;
+import lotto.domain.*;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+import java.util.List;
+import java.util.Map;
+
+public class LottoController {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final LottoMachine lottoMachine;
+
+    public LottoController(InputView inputView, OutputView outputView, LottoMachine lottoMachine) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.lottoMachine = lottoMachine;
+    }
+
+    public void run() {
+        String purchaseAmountInput = inputView.inputPurchaseAmount();
+        PurchaseAmount purchaseAmount = new PurchaseAmount(purchaseAmountInput);
+        int lottoCount = purchaseAmount.getLottoCount();
+        outputView.printLottoCount(lottoCount);
```

Comment:
```suggestion
        outputView.printLottoCount(purchaseAmount.getLottoCount());
```
이 정도는 바로 선언해도 가독성에 문제 없을 것 같아요!